.github/workflows/ci.yml

```diff
@@ -310,9 +310,8 @@ jobs:
             cache: ghcr.io/galoisinc/cache-saw-remote-api
     steps:
       - uses: actions/checkout@v2
-      - run: |
-          git submodule update --init
-          git -C deps/abcBridge submodule update --init
+        with:
+          submodules: true
```

Comment:
I think the original intent was to checkout _only_ the abcBridge dependency and not the  entire set of submodules, @ldettwy .